src/main/java/me/karakelley/tictactoe/BoardState.java

```diff
@@ -110,4 +114,11 @@ private int gridLength() {
     diagonals[1] = rightDiagonal();
     return diagonals;
   }
+
+  public List<Integer> availableCells() {
+    return IntStream.range(0, grid.length).
+            filter(i -> cellBlank(i)).
+            mapToObj(i -> i).
```

Comment:
1. Nice. Good use of Java 8 streams here.
1. What is this line doing?
1. Have a look at method references. Sometimes you can get cleaner-looking code using them rather than a lambda. `.filter(this::cellBlank)` instead of `.filter(i => cellBlank(i))`. Either way is perfectly fine.